### DISCUSSION TITLE
8-0 Part 2 Creating VRT for map of northern hemisphere

### DISCUSSION THREAD

**Discussion by vmwielgus** [ua-gist604b-f24/syllabus#49] [General]
While trying to create the VRT for the map of the northern hemisphere, I get an error saying the my -srcwin has negative width and/or height. 
My code:
`gdal_translate -of VRT -projwin 180 90 180 0 NE1_50M_SR_W_tenth.tif NE1_50M_SR_W_NH60.vrt`
I looked up the upper left x,y and lower left x,y of the northern hemisphere but obviously my degrees are wrong. Without necessarily giving away the answer, can someone explain how to modify the code to successfully create a VRT for the northern hemisphere? 
Thanks

**Comment by travisz09** [comment]
You are very close. Think carefully about your coordinates, and what they mean. Compare your answer to the coordinates for the S. Hemisphere.
[gdal_translate -projwin](https://gdal.org/en/stable/programs/gdal_translate.html#cmdoption-gdal_translate-projwin)

<details>
<summary>Hint</summary>
What is the difference between 180°E vs. 180°W?
</details>

<details>
<summary>Answer</summary>
-180 90 180 60
</details>

**Reply by vmwielgus** [reply]
Thank you so much Travis! I think I understand now.
